modules/openstack/openstack.go

```diff
@@ -117,7 +114,6 @@ func NewOpenStack(
 	if err != nil {
 		return nil, err
 	}
-
```

Comment:
nit: whitespace damage. Or is it go vet again? :)

Reply:
yes it was go vet

Reply:
then ignore me and chalk up one more to the "random acts of govet"